### ISSUE TITLE
bootstrap 3

### ISSUE THREAD

**Issue by Nabu-thinker-ru** [ppetrid/yawd-admin#22]
Are you planing to upgrade bootstrap? I need it, but i havent time for do it myself. Maybe i can spend some money for this?

**Comment by ghost** [comment]
Hello, upgrading to bootstrap would have been nice for yawd-admin, although it would introduce backwards-incompatible changes for projects that rely on the current yawd-admin templates. We do have plans for upgrading the app but it's not an easy process and there is no timeline for now. That being said, if you really need it quick and you'd like us to do the job for you, you could pm me at ppetrid@yawd.eu to discuss it further.

I'll leave this ticket open, that's something yawd-admin will need to move forwards. If other people in the community have opinions on how to proceed with the migration in a systematic way, I'd also like to hear them.

**Comment by Nabu-thinker-ru** [comment]
I don't believe in version-compatibles of django projects. All my projects have personal virtualenv with concrete version of django and other apps. And I can't upgrade anything without hard working. Thats why i think yawd-admin don't need backwards-compatibles.

**Comment by ghost** [comment]
I believe we can go for this. We'll create a bootstrap 3 branch, start working on the migration and see how it goes. As far as versions are concerned, once it's ready we'll explain people the new version will not work with previous template overrides and encourage them to freeze yawd-admin version in their requirements files if needed.

**Comment by esistgut** [comment]
news on this?

**Comment by Nabu-thinker-ru** [comment]
Nope :(
Maybe you wanna do this?
